### DISCUSSION TITLE
strongswan in docker ,after a few days of operation

### DISCUSSION THREAD

**Discussion by bugfan** [strongswan/strongswan#642] [General]
![image](https://user-images.githubusercontent.com/30284615/135194305-c19885af-6a89-42b6-9af6-3ff2da5e5d3a.png)

**Comment by bugfan** [comment]
docker's udp500 is binding,but inner ipsec is hang up

**Comment by strongX509** [comment]
Well, the error message when executing ipsec status indicates that the charon daemon is not running any more. Do you have any strongSwan logs?

**Comment by bugfan** [comment]
this problem only happened once, and several machines happened together. I wonder if I executed “chroot some dir”. There is no recurrence now. and i wonder if it is caused by "ipsrc start --nofork". It seems that the child process forked from the parent process is still running, but the parent process hangs
